EpikCord/__init__.py

```diff
@@ -1891,7 +1888,42 @@ def unload_section(self, section: Union[CommandsSection, EventsSection]):
 # class ClientGuildMember(Member):
 #     def __init__(self, client: Client,data: dict):
 #         super().__init__(data)
+class VoiceWebsocketClient:
+    def __init__(self, client:Client):
+        self.client = client
+    
+        
 
+    async def connect(self, guild_id:str, channel_id:str, **kwargs):
+        await self.client.send_json({
+            "op": 4,
+            "d" : {
+                "guild_id": guild_id,
+                "channel_id": channel_id,
+                "self_mute": kwargs.get("self_mute", False),
+                "self_deaf" : kwargs.get("self_deaf", False)
+            }
+        })
+        voice_state_update_response:WSMessage = await self.client.ws.receive()
+        voice_guild_update_response:WSMessage = await self.client.ws.receive()
+        self.session_id = voice_state_update_response.json()["session_id"]
+        self.guild_voice_data:dict = voice_guild_update_response.json()["d"]
+        await self.client.user.fetch() #update data
+        self.ws = await self.client.http.ws_connect("wss://"+ self.guild_voice_data["endpoint"])
+        await self.send_ws_json({
+            "op": 0,
+            "d" : {
+                "server_id": self.guild_voice_data["guild_id"],
+                "user_id": self.client.user.id,
+                "session_id": self.session_id,
+                "token": self.guild_voice_data["token"]
+            }
+        })
+
+    async def send_ws_json(self, json):
```

Comment:
Please rename this `send_json` to keep it consistent with the `Client`'s gateway websocket.